### PR TITLE
Change function name to _prime_post_parent_id_caches

### PR DESCRIPTION
**Pull request by spacedmonkey**
Trac ticket: https://core.trac.wordpress.org/ticket/59188

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/post.php [hunk 1]
```diff
@@ -7805,7 +7805,7 @@ function _prime_post_caches( $ids, $update_term_cache = true, $update_meta_cache
  *
  * @param int[] $ids ID list.
  */
-function _prime_post_parents_caches( array $ids ) {
+function _prime_post_parent_id_caches( array $ids ) {
```
Comment (joemcgill) [inline src/wp-includes/post.php:7808]: Can we bring over the other docblock improvements from @peterwilsoncc's PR ([here](https://github.com/WordPress/wordpress-develop/pull/5386/files#diff-ef9c5475561d459917dac57d21e08a6dfeb36f8b1c44b978d3696aec8b97c405R7811-R7814)) as well?

**Review by joemcgill — APPROVED** [review]
One suggestion, but pre-approving since they're docs updates. Not blocking.

File: tests/phpunit/tests/post/primePostParentIdCaches.php [hunk 2]
```diff
@@ -94,17 +94,17 @@ public function test_prime_post_parents_caches_update() {
 		);
 
 		$before_num_queries = get_num_queries();
-		_prime_post_parents_caches( array( $page_id ) );
+		_prime_post_parent_id_caches( array( $page_id ) );
 		$num_queries = get_num_queries() - $before_num_queries;
 
-		$this->assertSame( 1, $num_queries, 'Unexpected number of queries on second run' );
+		$this->assertSame( 0, $num_queries, 'Unexpected number of queries on second run' );
```
Comment (spacedmonkey) [inline tests/phpunit/tests/post/primePostParentIdCaches.php]:
```suggestion
		$this->assertSame( 1, $num_queries, 'Unexpected number of queries on second run' );
```

File: src/wp-includes/post.php (hunk 1 above)
Comment (spacedmonkey) [reply src/wp-includes/post.php:7808]: @joemcgill Done.

File: tests/phpunit/tests/post/primePostParentIdCaches.php (hunk 2 above)
Comment (joemcgill) [reply tests/phpunit/tests/post/primePostParentIdCaches.php]: Missed that this change had snuck in with the name change. 👍🏻

**Review by joemcgill — APPROVED** [review]
Re-approving, post updates.

**Review by peterwilsoncc — APPROVED** [review]
LGTM.

I can commit renaming the function while I have daylight so it makes it in to the next beta.

**Comment by peterwilsoncc** [comment]
Committed in [r56811](https://core.trac.wordpress.org/changeset/56811) / https://github.com/WordPress/wordpress-develop/commit/33b32ebc4ee8cdc577792c40ee47736394442dc4

**Pull request closed without merging**